crmsh/utils.py

```diff
@@ -26,6 +26,10 @@
 from .msg import common_warn, common_info, common_debug, common_err, err_buf
 
 
+class TerminateSubCommand(Exception):
+    pass
```

Comment:
This `pass` is not required. I would replace it by a normal docstring explaining what this exception is used for.
```
class TerminateSubCommand(Exception):
    '''
    This is a exception for...
    '''
```

Reply:
Thanks @arbulu89 !
Changed!